### DISCUSSION TITLE
Kaspa coin

### DISCUSSION THREAD

**Discussion by sacredland** [Lolliedieb/lolMiner-releases#1636] [General]
Do you plan to add kaspa coin to lolminer?   As bzminer did and it’s mostly your biggest competitor.. ;)

i honestly hope so.

**Comment by ljburna** [comment]
Same here since the latest version does not support kaspa although it say it does.


![Selection_001](https://user-images.githubusercontent.com/113790414/190867162-2881448f-7afc-4b0d-b692-b0e98f08a03f.png)
![Selection_002](https://user-images.githubusercontent.com/113790414/190867163-32f93ed4-f664-4847-9648-376c6696c854.png)
![Selection_003](https://user-images.githubusercontent.com/113790414/190867164-0180660b-bd3e-4587-8191-048147a5647b.png)
